tests/json-fixtures/test_state.py

```diff
@@ -295,6 +302,8 @@ def test_state_fixtures(fixture, fixture_vm_class):
         computation, _ = vm.apply_transaction(transaction)
     except ValidationError as err:
         transaction_error = err
+        LOGGER.warn("Got transaction error:")
+        LOGGER.warn(traceback.format_exc())
```

Comment:
Just FYI

```python
logger.warn("Got transaction error", exc_info=True)
```

This can be used anywhere inside an `except` block to include a traceback of the exception in the log message (at whatever log level you specify).

Reply:
Good to know, thanks!